app/src/main/AndroidManifest.xml

```diff
@@ -194,6 +195,10 @@
       android:name=".app.walkthrough.WalkthroughActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity android:name=".app.testing.LessonThumbnailImageViewTestActivity" />
+    <activity
+      android:name=".app.devoptions.DeveloperOptionsActivity"
+      android:label="@string/developer_options_activity_title"
```

Comment:
Please add a test case for this label similar to `testFaqListActivity_hasCorrectActivityLabel`

Reply:
Done. PTAL.